col.go

```diff
@@ -312,7 +313,7 @@ func (c *Column) Resize(r image.Rectangle) {
 }
 
 func (c *Column) Sort() {
-	sort.Slice(c.w, func(i, j int) bool { return c.w[i].body.file.name < c.w[j].body.file.name })
+	sort.Slice(c.w, func(i, j int) bool { return c.w[i].body.file.details.name < c.w[j].body.file.details.name })
```

Comment:
This line is either a rebase mistake or #335 as the first CL is not the right order. What's your preference here? 🙂

Reply:
My bad, I forgot that I had rebased it back in, before our conversation about the branches depending on eachother and the headache that causes.